src/indev/lv_indev_scroll.c

```diff
@@ -500,17 +548,18 @@ static int32_t find_snap_point_x(const lv_obj_t * obj, int32_t min, int32_t max,
         }
     }
 
-    return dist == LV_COORD_MAX ? 0 : -dist;
+    return dist == LV_COORD_MAX ? LV_COORD_MAX : -dist;
```

Comment:
This function has another `return` in it. Should the other `return` be updated to return `LV_COORD_MAX` too? FYI, that other `return` -- taken when `align == LV_SCROLL_SNAP_NONE` -- will only be reached when this function is called by the public function `lv_indev_scroll_get_snap_dist`.

Reply:
Added.